pkg/sabactl/cmd/machines.go

```diff
@@ -191,19 +209,32 @@ var machinesSetRetireDateCmd = &cobra.Command{
 
 func init() {
 	getOpts := map[string]string{
-		"serial":   "Serial name",
-		"rack":     "Rack name",
-		"role":     "Role name",
-		"labels":   "Label name and value (--labels key=val,...)",
-		"ipv4":     "IPv4 address",
-		"ipv6":     "IPv6 address",
-		"bmc-type": "BMC type",
-		"state":    "State",
+		"serial":           "Serial name",
+		"rack":             "Rack name",
```

Comment:
Options that allow multiple specifications should be named in the plural form like `racks`.
You should also add an explanation ("Rack names (--racks 1,2,3...)").

If you don't want to break compatibility, you can provide two options, like `rack` and `racks`.

Reply:
Let's discuss if we can break compatibility.